test/first-order/no-arguments.coffee

```diff
@@ -64,16 +64,29 @@ module.exports = async ->
 
   run "sass"
 
+  define "handlebars", async ->
+    yield go [
+      glob "*.{hb,hbs,handlebars}", src
```

Comment:
I may be "preaching to the choir", but it wasn't obvious to me at first:
FWIW, h9 usage will override this glob. You could glob *.cookies here, and it would have no impact on the glob h9 uses.

For this reason, I only tested `*.scss` in the [sass PR](https://github.com/pandastrike/panda-9000/pull/11/files#diff-959548599c2d067b69cd4493bc838b9aR58), versus testing `*.{scss,sass}`.

I don't see a problem with testing all 3 extensions in P9K, but I also don't see how it adds any value.